### DISCUSSION TITLE
Help with onRowClick

### DISCUSSION THREAD

**Discussion by adarshSrivastava01** [TanStack/table#5329] [Q&A]
Hey TanStack Community,

is there a way I can do `<TanStackTable onRowClick={rowData => callSomeFunction(rowData.id, rowData.name)} />`

**Comment by goatfryed** [comment]
To my knowledge, @tanstack/table is headless and doesn't provide a TanStackTable component. Is this one of your project's components? 

You can make this work, but this seems to be a general question and not a library specific question better asked in communities for general react development advice.
Also, consider to provide more context, when you ask questions there.
